tests/phpunit/PropertySuggester/GetSuggestionsTest.php

```diff
@@ -7,7 +7,14 @@
 use Wikibase\DataModel\Entity\EntityId;
 use Wikibase\DataModel\Entity\Property;
 use Wikibase\Repo\WikibaseRepo;
-use Wikibase\Test\Repo\Api\WikibaseApiTestCase;
+use Wikibase\Repo\Tests\Api\WikibaseApiTestCase;
+
+// FIXME: Remove this temporary workaround after the Wikibase class was moved.
+if ( !class_exists( WikibaseApiTestCase::class ) ) {
```

Comment:
What do you mean with "this"? I imported the new (not yet existing) class name on purpose. This means all I have to do later is removing the `if`, but the import section above is already in it's final state.